src/flakeref.rs

```diff
@@ -575,6 +103,46 @@ impl std::str::FromStr for FlakeRef {
     }
 }
 
+#[cfg(test)]
+mod tests_parsers {
```

Comment:
If you already have a descriptive name, like `parsers` I don't think the `tests_` prefix adds too much, since it is already marked as test. This keeps the test output a little more condensed :).

Reply:
My thinking is that the "default" is `test` and `parsers` appends to that. You have your general tests, then you have your parser tests.

The utility is so that you can test just parsers with `cargo test pars`, for example. 



I don't want to risk bike shedding this at this point. If it's up to me, I'll keep going, so long as it continues to be trivially easy to make the changes you ask for. As I type this, I'm assuming that I'll have to do it at some point anyway.

Reply:
Agreed, it's just a nit. Do what is more comfortable for you :).